Docs: replacing "setting" to "option" in world docs (#2622)
ArchipelagoMW/Archipelago: worlds/noita/docs/en_Noita.md

# Noita

## Where is the options page?

The [player options page for this game](../player-options) contains all the options you need to configure and export a
config file.

## What does randomization do to this game?

Noita is a procedurally generated action roguelike. During runs in Noita you will find potions, wands, spells, perks,
chests, etc. Shop items, chests/hearts hidden in the environment, and pedestal items will be replaced with location

> checks. Orbs and boss drops can give location checks as well, if they are enabled in the options.

Reviewer: ```suggestion
checks. Orbs and boss drops can give location checks as well, if their respective options are enabled.
```

Author: That is a better phrasing, definitely
